docs/src/inscriptions/offset.md

```diff
@@ -0,0 +1,48 @@
+Offset
+======
+
+In order to make an inscription on a sat other than the first of its input, an
+offset can be provided with tag `2`, causing the inscription to be made on the
+sat at the given offset. If the offset is equal to or greater than the number
+of total sats in the outputs of the inscribe transaction, it is ignored, and
+the inscription is made as usual. The value of the offset tag is the little
+endian offset, with trailing zeros omitted.
+
+An even tag is used, so that old versions of `ord` consider the inscription to
+be unbound, instead of assigning it, incorrectly, to the first sat.
+
+This can be used to create multiple inscriptions in a single transaction on
```

Comment:
unclear what the mechanism is to create multiple inscriptions. To make `n` inscriptions do you need `n` inputs with an inscription envelope in the witness of each one? Can you put multiple inscriptions in the same input?

Reply:
you can do either. At the moment inscriptions in different inputs assigned to first sat of their respective input and inscriptions in same input are made unbound (they will be made reinscriptions on first sat of input soon)